### ISSUE TITLE
TargeSdkVersion should be 26 or higher

### ISSUE THREAD

**Issue by iadeelzafar** [opendatakit/share#119]
Currently, the targetSdkVersion is 22 which is no longer supported.

Here's the lint error.

![screenshot_2](https://user-images.githubusercontent.com/36811908/53297478-236ebb80-3841-11e9-88c2-0b0deea9d334.png)

* I think we should update it to 28.
* After updating it to 28, we will also have to update the compileSdkVersion to 28 as well.
@lakshyagupta21 @shobhitagarwal1612 What you think?

**Comment by jddeep** [comment]
@lakshyagupta21 @shobhitagarwal1612 I would like to work on this issue. Shall I?

**Comment by jddeep** [comment]
@lakshyagupta21 Please assign it for me. Thanks

**Comment by iadeelzafar** [comment]
@jddeep This issue needs discussion.

**Comment by jddeep** [comment]
@iadeelzafar so let's discuss then.

**Comment by jddeep** [comment]
Let's bring in mentors as well.

**Comment by iadeelzafar** [comment]
@jddeep Wait for the collaborators/maintainers to look upon this issue.

**Comment by shobhitagarwal1612** [comment]
Thanks for raising the issue @iadeelzafar 

Updating to targetSdkVersion 28 also means that we will have to ask for the [runtime permissions](https://developer.android.com/training/permissions/requesting)

**Comment by jddeep** [comment]
@shobhitagarwal1612 I would like to work on this.

**Comment by iadeelzafar** [comment]
@shobhitagarwal1612 yeah. Google Play requires new apps to Target API 26 or higher. So, which API do you think would the best to Target?

**Comment by jddeep** [comment]
I think it should be 26

**Comment by shobhitagarwal1612** [comment]
Sounds good to me. 
@iadeelzafar Since you are already working on another issue, can you let @jddeep take this one on?

**Comment by iadeelzafar** [comment]
Sure. @jddeep Go ahead. Good Luck!

**Comment by iadeelzafar** [comment]
@shobhitagarwal1612 But you didn't answered which API we should target? 26?

**Comment by shobhitagarwal1612** [comment]
API 26 should be fine for now

**Comment by iadeelzafar** [comment]
Alright.
@jddeep You can type ``` @opendatakit-bot claim ``` to claim this issue.

**Comment by iadeelzafar** [comment]
@shobhitagarwal1612 How may I un-assign myself?

**Comment by jddeep** [comment]
Okay then I will be working on the issue. Thanks guys.

**Comment by shobhitagarwal1612** [comment]
Please claim the issue then @jddeep 

Also before you start coding, Please list down the tasks that need to be performed in order to solve this issue here in this thread.

**Comment by jddeep** [comment]
@shobhitagarwal1612 please assign me the issue. Thanks

**Comment by iadeelzafar** [comment]
@jddeep read my comment above.

**Comment by jddeep** [comment]
@opendatakit-bot claim

**Comment by jddeep** [comment]
@opendatakit-bot claim --force

**Comment by udhay24** [comment]
@opendatakit-bot claim

**Comment by udhay24** [comment]
@shobhitagarwal1612 @lakshyagupta21 what should be expected behaviour when the user denies storage permission?

**Comment by jddeep** [comment]
@shobhitagarwal1612 I am extremely sorry for the delay. Actually, I am stuck in some other problems right now and am not able to give enough time to my OS contributions. Hope you don't mind. I will surely get back to the org's projects as soon as I get out of this. Thanks.

**Comment by udhay24** [comment]
![mobizen_20190315_194626_1](https://user-images.githubusercontent.com/29946262/54438618-0270fa80-475d-11e9-9bc3-26a526d64a84.gif)

asking runtime storage permission in Oreo 8.1
